packages/api/src/Base.ts

```diff
@@ -474,6 +476,10 @@ export default abstract class ApiBase<ApiType> {
           this.emit('ready', this);
         }
 
+        if (!Object.keys(this.consts).length) {
+          this._isSubstrateV2 = false;
```

Comment:
I believe this is the other way around?

Reply:
It's maybe a little hard to read, but if the length is 0 it's a v1, the `!` will make it go into the `if` to set it to `false`.

Reply:
You are right :)